openex-api/src/main/java/io/openex/injects/email/service/EmailPgp.java

```diff
@@ -23,57 +23,57 @@
 @Component
 public class EmailPgp {
 
-    public PGPPublicKey getUserPgpKey(User user) throws IOException {
```

Comment:
Bingo ! It seems that we had a problem on the sending an email with a pgp key too